src/KNNClassifier/index.js

```diff
@@ -31,20 +33,33 @@ class KNN {
     } else if (classIndexOrLabel === 'number') {
       classIndex = classIndexOrLabel;
     }
+
+    if (Array.isArray(input)) {
+      example = tf.tensor(input);
```

Comment:
Does this work fo a multi-dimensional array. Or do we need to flatten or manage the shape somehow? We might need some error handling if a user passes in multiple examples with different shapes (although perhaps tf.js handles this somehow.)

Reply:
This works for muti-dimensional array, it accepts a nested array of numbers, or a flat array([tf.tensor](https://js.tensorflow.org/api/0.14.1/#tensor)). We can also accept an optional `shape` in the future?

tf.js knn classifier handles the different shapes: https://github.com/tensorflow/tfjs-models/blob/master/knn-classifier/src/index.ts#L47.